set.mm

```diff
@@ -98749,7 +98753,7 @@ our Axiom of Choice (in the form of ~ ac2 ).  The proof does not depend
 
   $( Cardinal addition of equinumerous sets.  Exercise 4.56(b) of [Mendelson]
      p. 258.  (Contributed by NM, 28-Sep-2004.)  (Revised by Mario Carneiro,
-     29-Apr-2015.) $)
+     29-Apr-2015.)  Use ~ djuen instead.  (New usage is discouraged.) $)
```

Comment:
If ~cdaen will be removed soon, the comment should be aligned with the comment of ~djuen. Especially the original contributor should be kept (at least if the proofs are still similar). Proposal for a comment of ~djuen:

`$( Disjoint unions of equinumerous sets are equinumerous.  This corresponds to the cardinal addition of equinumerous sets.  Exercise 4.56(b) of [Mendelson] p. 258.  (Contributed by NM, 28-Sep-2004.)  (Revised by Mario Carneiro, 29-Apr-2015.) (Revised by Jim Kingdon, 30-Jul-2023.) $)`

Reply:
This also holds for the other theorems, of course.

Reply:
See ~dju1dif  how it should be (its comment corresponds to my proposal).

Reply:
As for the comments in general, this runs right into the "it depends" nature of our conventions.  There is no convention to preserve existing `Revised` lines (in general). There is a convention to (generally speaking, not always) avoid more than one `Revised` line. If it were just up to me, in the vast majority of cases I'd list the first to contribute in the `Contributed by` comment and last to contribute in a single `Revised`, and that's it. But I realize there is a decent amount of sentiment for applying more subtlety than that.

I don't much care whether there is a `Revised` line for my changes, and given the mostly mechanical nature of most of them I'm not sure there's an especially strong objective case for it.

Now, if there is no existing `Revised` line the concerns about clutter are presumably less (although we might just be delaying the question until the next time someone revises it, so I'd be fine with not adding a `Revised` line there either).

As for djuen in particular, that looks like an oversight. I looked over the pull request and this seems to affect `djuen` and `endjudisj` so I pushed up changes to the comments for those two.